doc/build.md

```diff
@@ -57,18 +57,15 @@ properly configure it for xu4, build `libboron.a`, and install it:
 
     make -C dist boron
     make -C dist/boron libboron.a
-    sudo make -C dist/boron DESTDIR=/usr install-dev
-    sudo make -C dist/boron DESTDIR=/usr install
+    sudo make -C dist/boron DESTDIR=/usr install install-dev
 
 Faun SDKs and source can be downloaded from the
 [Faun Releases](https://github.com/WickedSmoke/faun/releases) page.  The
 source is also available via the src/faun Git submodule:
 
-    git submodule init
-    git submodule update
-    ./src/faun/configure --prefix /usr
+    git submodule update --init --recursive
```

Comment:
The --recursive option isn't needed here (and is misleading about what the repo. contains).